Docs IA 2.0: Improve landing page, add intro to Getting Started page (#78765)
vercel/next.js: docs/index.mdx

description: Welcome to the Next.js Documentation.
related:
  title: Next Steps
  description: Create your first application and learn the core Next.js features.
  links:
    - app/getting-started
---

Welcome to the Next.js documentation!

## What is Next.js?

> Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations like routing, server rendering, and more.

Reviewer: ```suggestion
Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations.
```

For optimizations I was thinking more like images, fonts, scripts, etc – but my main there here is not wanting to drop too many terms, too early.

It also handles configuration for bundlers and compilers, so you can focus on building your application-not managing tooling.

Whether you're an individual developer or part of a larger team, Next.js can help you build interactive, dynamic, and fast React applications.

## How to use the docs

The docs are organized into 4 sections:

- [Getting Started](/docs/app/getting-started): Step-by-step tutorials to help you create a new application and learn the core Next.js features.
- [Guides](/docs/app/guides): Tutorials on specific use cases, choose what's relevant to you.
- [Deep Dive](/docs/app/deep-dive): In-depth explanations on how Next.js works.
- [API Reference](/docs/app/api-reference): Detailed technical reference for every feature.

> Use the sidebar to navigate to navigate through the sections, or search (`Ctrl+K` or `Cmd+K`) to quickly find a page.

Reviewer: ```suggestion
Use the sidebar to navigate through the sections, or search (`Ctrl+K` or `Cmd+K`) to quickly find a page.
```

## App Router and Pages Router

Next.js has two different routers:

> - **App Router**: The newer router that supports React's latest features, such as Server Components and Streaming.

Reviewer: ```suggestion
- **App Router**: The newer router that supports new React features like Server Components.
```

> - **Pages Router**: The original router, still supported for older applications.

Reviewer: ```suggestion
- **Pages Router**: The original router, still supported and being improved.
```

This clarity could be helpful, e.g. Turbopack landing for Pages Router.

At the top of the sidebar, you'll notice a dropdown menu that allows you to switch between the [App Router](/docs/app) and the [Pages Router](/docs/pages) docs.

## Pre-requisite knowledge

> Our documentation is designed to be beginner-friendly, but we assume some familiarity with web development. Before getting started, it'll help if you're comfortable with:

Reviewer: ```suggestion
Our documentation assumes some familiarity with web development. Before getting started, it'll help if you're comfortable with:
```

- HTML
- CSS
- JavaScript
- React

> If you're new to React or need a refresher, we recommend starting with our [React Foundations course](/learn/react-foundations). We also recommend the [Next.js Foundations course](/learn/dashboard-app) that has you building an application as you learn.

Reviewer: Might be worth adding this on the other page as well

Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations.

> It also handles configuration for bundlers and compilers, so you can focus on building your application-not managing tooling.

Reviewer: ```suggestion
It also automatically configures lower-level tools like bundlers and compilers. You can instead focus on building your product and shipping quickly.
```
